projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php

```diff
@@ -104,7 +104,7 @@ public function should_disable_links_manager() {
 	 * Adds My Home menu.
 	 */
 	public function add_my_home_menu() {
-		$this->update_menu( 'index.php', 'https://wordpress.com/home/' . $this->domain, __( 'My Home', 'jetpack' ), 'manage_options', 'dashicons-admin-home' );
+		$this->update_menu( 'index.php', 'https://wordpress.com/home/' . $this->domain, __( 'My Home', 'jetpack' ), 'edit_posts', 'dashicons-admin-home' );
```

Comment:
I wonder if we should use a Jetpack capability here instead of a core one? If My Home includes stats for example, how do we ensure that those stats are only visible to folks with stats access on a Jetpack site, since Jetpack allows you to define what roles have access to stats? 
Unless the "My Home" page content is dynamic based on your role and capabilities?

Reply:
> Unless the "My Home" page content is dynamic based on your role and capabilities?

This. In D71149-code we made sure that the content visible in My Home is aligned with the capabilities of the user. For instance, the Stats card is hidden if the user cannot view them (we check that with the `stats_is_blog_user` function).